### PR TITLE
aktualizr-lite: Add doc on aklite conf params

### PR DESCRIPTION
**Pull request by mike-sul**
Signed-off-by: Mike Sul <mike.sul@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 32e3630941f926a6c975b950a38b86238fe03ac0 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/637/docs/artifacts/html/index.html

File: source/reference/aktualizr-lite.rst [hunk 1]
```diff
@@ -69,3 +69,48 @@ Apply Specific Update
...
+The following are aktualizr-repo's configuration parameters that can be useful to play with, the presented values are the default one.
+
+.. code-block::
+
+    [uptane]
+    # Target/Update check-in interval
+    polling_sec = 300
+
+    [pacman]
+    # A list of Compose Apps to update
```
Comment (doanac) [inline source/reference/aktualizr-lite.rst:95]: might mention this is a comma separated list.

**Review by doanac — COMMENTED** [review]
Looks good minus one nit-pick.

@mike-scott - is this fine for merging into master, or should we just land this in the refactor branch instead to make things easier?

File: source/reference/aktualizr-lite.rst (hunk 1 above)
Comment (mike-sul) [reply source/reference/aktualizr-lite.rst]: updated

**Comment by doanac** [comment]
Docs for 34e86959a7e86c5951f170b6bb70afd9aff9d857 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/638/docs/artifacts/html/index.html

**Review by doanac — APPROVED** [review]

**Comment by doanac** [comment]
you may want to wait until monday. I think master is about to get changed quite a bit.

**Comment by doanac** [comment]
Docs for be511fe8dc83457428fec21db7d436db9db6d5a7 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/640/docs/artifacts/html/index.html

**Comment by mike-sul** [comment]
@doanac  rebased on the latest master, is it good to go?

**Comment by doanac** [comment]
@mike-sul - it doesn't look like the refactor branch has been merged into master yet. Lets see when @MatthewCroughan merges and then try.

**Comment by doanac** [comment]
Docs for 95fde53598f98c64b1e02637f55b44ac6b1ee54a are browesable at: https://ci.foundries.io/projects/fio-docs/builds/648/docs/artifacts/html/index.html

**Comment by mike-sul** [comment]
@doanac rebased on master.

**Review by doanac — APPROVED** [review]

**Pull request merged**